### PR TITLE
Add Quarkus as official blueprint

### PR DESCRIPTION
**Pull request by avdev4j**
https://github.com/jhipster/generator-jhipster-quarkus/issues/71

### CODE REVIEW

**Review by mraible — APPROVED** [review]

File: modules/official-blueprints.md [hunk 1]
```diff
@@ -37,13 +37,17 @@ Stream lead is: [Daniel Petisme](https://github.com/danielpetisme).
 
 ## Node.js
 
-Latest official blueprint, it replaces the server java side with the NestJS Node framework. Source code and documentation could be found here: [repository](https://github.com/jhipster/generator-jhipster-nodejs).
+Fourth official blueprint, it replaces the server java side with the NestJS Node framework. Source code and documentation could be found here: [repository](https://github.com/jhipster/generator-jhipster-nodejs).
```
Comment (joewhite101) [inline modules/official-blueprints.md:40]: Numbering these probably isn't a good idea. Will be a maintenance headache and doesn't adding anything. I think just a general description for each is good.

Removing latest is a good idea but i don't think the ordering is needed.

**Review by joewhite101 — CHANGES_REQUESTED** [review]
Small change

File: modules/official-blueprints.md (hunk 1 above)
Comment (avdev4j) [reply modules/official-blueprints.md:40]: I wonder the same, to be honest. We should not add the idea there is a kind of order. I'll remove all this numbers.

**Review by DanielFran — APPROVED** [review]
LGTM

**Pull request merged**